@here/harp-geoutils/lib/coordinates/GeoCoordinates.ts

```diff
@@ -154,4 +154,30 @@ export class GeoCoordinates implements GeoCoordinatesLike {
     clone(): GeoCoordinates {
         return new GeoCoordinates(this.latitude, this.longitude, this.altitude);
     }
+
+    /**
+     * Update this `GeoCoordinates` by assigning it the southwesternmost position between it and
+     * another `GeoCoordinates`.
+     *
+     * @param other The other `GeoCoordinates` to compare to this one.
+     */
+    min(other: GeoCoordinates) {
+        this.latitude = Math.min(this.latitude, other.latitude);
+        this.longitude = Math.min(this.longitude, other.longitude);
+        this.altitude =
+            this.altitude && other.altitude ? Math.min(this.altitude, other.altitude) : undefined;
```

Comment:
Might be a bit counter-intuitive if this has an altitude and other has none that the result will also have none.
In general this method does not make much sense as a member of GeoCoordinates but only in combination with GeoBox, so consider to move the code there.

Reply:
Maybe it could be a util method which returns the combined min of two GeoCoordinates